### PR TITLE
fix(HMSPROV-390): unscoped update pubkey fix

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
I guess you meant to use this in migrations?

**Comment by lzap** [comment]
Damn it is not used :-))))

**Comment by lzap** [comment]
I was rushing it because of a meeting, fixed, rebased.

**Comment by lzap** [comment]
Alternatively, I could simply write SQL in the migration itself, which seems more appropriate I think?

**Comment by lzap** [comment]
I have a better idea, do not merge!

**Comment by ezr-ondrej** [comment]
Let's remove https://github.com/RHEnVision/provisioning-backend/blob/f5b2e1992b1987bdb7c95685f7ae2e607a420990/internal/migrations/code/update_fingerprints_test.go#L19 from the tests, as we do not have that in prod code :)

**Comment by lzap** [comment]
Good point, I cannot remove the context with account because tests need to create testing records, but I am introducing new context that is only used for the actual test (the function that is being tested). That must not have account, others do require it. Hope it makes sense.

File: internal/migrations/code/update_fingerprints_test.go [hunk 1]
```diff
@@ -15,12 +15,18 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func newContextWithAccount(t *testing.T) context.Context {
+	return identity.WithTenant(t, context.Background())
+}
+
 func newContext(t *testing.T) context.Context {
 	return identity.WithTenant(t, context.Background())
```
Comment (ezr-ondrej) [inline internal/migrations/code/update_fingerprints_test.go:23]: We have the context here as well, is that intentinal? I thought we wanted clean context for this one? :thinking:

Comment (lzap) [reply internal/migrations/code/update_fingerprints_test.go:23]: DAMN

**Comment by lzap** [comment]
Rebased OMG when one's in a rush, this is the end result.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @lzap ! :+1:

**Pull request merged**